app/admin/epic_admin.rb

```diff
@@ -0,0 +1,15 @@
+Trestle.resource(:epic, model: Epic) do
```

Comment:
I think that you don't have to put `model: Epic` when you use same model and resource name